src/Resources/translations/messages.fr.yml

```
@@ -62,21 +62,21 @@ sylius:
                     title: Catégories
                     subtitle: Configuration des catégories
                     categories_to_exclude: Catégories à exclure
-                    main_category: Catégorie principal
+                    main_category: Catégorie principale
                     root_categories: Catégories racines de Sylius
                     empty_local_replace_by: Remplacer la locale vide par
                 products:
                     title: Produits
                     subtitle: Configuration des produits
-                    akeneo_price_attribute: Attribue Akeneo du prix
+                    akeneo_price_attribute: Attribut Akeneo du prix
                     akeneo_enabled_channels_attribute: Attribut Akeneo des channels actifs
                     attribute: Attribut
                     value: Valeur
                     import_media_files: Import de média
                     akeneo_attributes: Attributs Akeneo
                     sylius_attribute: Attribut Sylius
                     akeneo_attribute: Attribut Akeneo
-                    regenerate_url_rewrites: Regenerate url rewrites
+                    regenerate_url_rewrites: "Régénérer les réécritures d'url"
```

I agree that the english sentence is not the best too :sweat_smile: 

```suggestion
                    regenerate_url_rewrites: "Autoriser la modification des URL"
```